Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1083,9 +1085,11 @@ public function toJson()
      */
     public function __clone()
     {
-        foreach ($this as $key => $val) {
-            $this[$key] = clone $val;
+        foreach ($this as $key => &$val) {
+            $val = clone $val;
         }
+        // unset reference to keep scope clear
+        unset($val);
```

Comment:
Foreach doesn't create own scope, so both references and index remain and as functions grows side-effects/miss-usages possible. Better to keep.

In general array_keys replaced with unset, bur benefit is loop performance and memory usage.